Libplanet.Net/Consensus/ConsensusReactor.cs

```diff
@@ -112,6 +113,42 @@ private async Task ProcessMessageHandler(Message message)
                     await ReplyMessagePongAsync(message);
                     _consensusContext.HandleMessage(consensusMessage);
                     break;
+                case Ping ping:
+                    await ReplyMessagePongAsync(ping);
+                    break;
+            }
+        }
+
+        private async Task CheckValidatorsLiveness(CancellationToken ctx)
+        {
+            while (!ctx.IsCancellationRequested)
+            {
+                var tasks = _validatorPeers
+                    .Select(peer => _consensusTransport
+                        .SendMessageAsync(peer, new Ping(), TimeSpan.FromSeconds(1), ctx))
+                    .ToList();
+                try
+                {
+                    await Task.WhenAll(tasks);
+                }
+                catch (AggregateException)
+                {
+                }
```

Comment:
Is it intended? (or wip?)

Reply:
Intended. I want to just catch `AggregateException` and ignore.